server/command.go

```diff
@@ -241,18 +241,40 @@ func (p *Plugin) runConnectCommand(user *model.User, extra *model.CommandArgs) (
 }
 
 func (p *Plugin) runSubscribeCommand(user *model.User, extra *model.CommandArgs, meetingID int) (string, error) {
+	if !p.API.HasPermissionToChannel(user.Id, extra.ChannelId, model.PermissionCreatePost) {
```

Comment:
This is a product question - Who should have permissions to subscribe a channel to a meeting id? Probably good to DRY it up into its own method as well

Reply:
I think a bit of repetition here is not that bad, anyway, I think the way of thinking about it, for me is, if you are able to publish a post, you will be able to do it anyway, but I'm ok if we only allow this to channel admins, for example.